LDMP/calculate.py

```diff
@@ -714,6 +756,76 @@ def check_overwrites(self):
         return True
 
 
+class CalculationHidedOutputWidget(QtWidgets.QWidget, Ui_WidgetCalculationOutput):
```

Comment:
this class leave old CalculationOutputWidget unchanged for retro compatibility reasons and added a copy of the class tab where out names are set programmatically